MovieQuiz/App/APIKey.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  APIKey.swift
+//  MovieQuiz
+//
+//  Created by SERGEY SHLYAKHIN on 18.11.2022.
+//
+
+import Foundation
+
+let apiKey = "k_nqxtwh99"
```

Comment:
Принято глобальный контекст оборачивать в enum/scruct. В swift нет аналога неймспейсов c++, а enum-ы позволяют такое проделывать. 

Глобальные константы не привествуются

```suggestion
enum Secrets {
    static let apiKey = "k_nqxtwh99"
}
```

Reply:
Сделаю. Не думал, что вы запускаете наши проекты) . В своих проектах будут закрывать файл через .gitignore

Reply:
Если умеешь, то круто, да, так стоит делать всегда (с небольшой поправкой, что ревьювер может не иметь ключа :)